crates/ruff_linter/src/rules/flake8_unused_arguments/rules/unused_arguments.rs

```diff
@@ -461,6 +458,19 @@ pub(crate) fn unused_arguments(checker: &Checker, scope: &Scope) {
                         function(Argumentable::StaticMethod, parameters, scope, checker);
                     }
                 }
+                function_type::FunctionType::NewMethod => {
+                    if checker.enabled(Argumentable::StaticMethod.rule_code())
+                        && !function_type::is_stub(function_def, checker.semantic())
+                        && !is_not_implemented_stub_with_variable(function_def, checker.semantic())
+                        && !visibility::is_abstract(decorator_list, checker.semantic())
+                        && !visibility::is_override(decorator_list, checker.semantic())
+                        && !visibility::is_overload(decorator_list, checker.semantic())
+                    {
+                        // we use `method()` here rather than `function()`, as although `__new__` is
+                        // an implicit staticmethod, `__new__` methods must always have >= parameter
```

Comment:
Ah shoot, my previous suggestion had a typo -- it should have been this :-(

```suggestion
                        // an implicit staticmethod, `__new__` methods must always have >=1 parameters
```